versioned_docs/version-latest/protocol/security.md

```diff
@@ -1,86 +1,266 @@
 import Link from '@docusaurus/Link';
 import styles from '@site/src/pages/index.module.css';
 
-Navigating the internet and being part of a crypto project can be exciting! But it's crucial to use good security practices. This way, we can protect our digital community from cyber attacks, and enjoy our Subspace journey with peace of mind.
+Navigating the internet and being part of a crypto project can be exciting! But it's crucial to use best security practices. Here is a general guide to security and safety advice, including basic measures for securing servers and home PCs. This is your contribution to keeping Subspace network up and running and of course the safety of the community's cryptoassets. 
 
+Enjoy Subspace journey with peace of mind!
 
-## Wallet security
 
----
+# Wallet security
 
-:::danger NEVER SHARE YOUR KEYS
+### Passwords
+*
+ Be sure to make the password long enough, combining both registers letters, numbers and special characters. An 11-character password increases the bruteforce attack time to normally impossible limit but please take a look at [passwords brutforce time illustrating table](https://www.techrepublic.com/wp-content/uploads/2022/03/tr8423-Figure-A-How-an-8-character-password-could-be-cracked-in-just-a-few-minutes-770x404.jpg). However, even a complex password is not all, that is required for proper protection. This will be discussed in this article.
+*
+ Avoid using common or easily guessed combinations of words or letters, as well as parts of your online data or reusing any of your existing passwords.
 
-Under any circumstances, do not share your 12-word seed phrase or private keys.
-:::
+Weak password example:
+```
+S0methin9C00l!
+```
+Strong password example:
+```
+^p$O_~a!4h{G'9C*
+```
+*
+ Consider using a password manager to generate complex passwords and store them securely. There are plenty of password managers available. Ensure that your choice stores data in encrypted form, boasts a robust password generator, receives positive reviews, and has a history of consistent updates. **For example, well-regarded options include KeePass, Bitwarden, 1Password, LastPass, Dashlane, and Keeper.**
+ However, it's crucial to do your own research and consult ratings before making a decision.
+*
+ Do not share your 12-word seed phrase or private keys.
+*
+ A hardware wallet is by far the most secure way to store your private keys. Unlike online exchanges and wallets, hardware wallets store the keys offline and protect from potential malware or hacking attempts.
+The most popular hardware wallets are [Ledger](https://www.ledger.com/) and [Tresor](https://trezor.io/).
+*
+ Backup your wallets: store the backups securely offline or in encrypted cloud storage. This ensures hat you can recover your funds in case of device loss, damage, or theft. There are also many encrypted cloud storage services available, like Tresorit, pCloud, Sync.com, SpiderOak, Mega (order does not matter). But always do your own research.
```

Comment:
```suggestion
 Backup your wallets: store the backups securely offline or in encrypted cloud storage. This ensures that you can recover your funds in case of device loss, damage, or theft. There are also many encrypted cloud storage services available, like Tresorit, pCloud, Sync.com, SpiderOak, Mega (order does not matter). But always do your own research.
```